packages/deployments/contracts/contracts/messaging/connectors/axelar/BaseAxelar.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.17;
+
+import {AxelarExecutable} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/executable/AxelarExecutable.sol";
+import {IAxelarGateway} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol";
+import {IAxelarGasService} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGasService.sol";
+import {StringToAddress, AddressToString} from "@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/AddressString.sol";
+
+abstract contract BaseAxelar is AxelarExecutable {
+  using StringToAddress for string;
+  using AddressToString for address;
+
+  // ============ Internal Storage ============
+  IAxelarGasService public immutable gasService;
```

Comment:
will the address ever change?

Reply:
https://etherscan.io/address/0x2d5d7d31F671F86C782533cc367F14109a082712#code

GasService contract implemented using proxy, so I don't think it will not change